### PR TITLE
gh-112007: Re-organize help utility intro message

### PR DESCRIPTION
**Pull request by terryjreedy**
Most important: move how-to-quit sentence to the end and mention 'q'.
Re-group the other sentences and improve some wording.

* Issue: gh-112007

### CODE REVIEW

File: Lib/pydoc.py [hunk 1]
```diff
@@ -2074,19 +2074,21 @@ def help(self, request, is_cli=False):
...
+Each module also comes with a one-line summary of what it does; to list
+the modules whose name or summary contain a given string such as "spam",
+type "modules spam".
 
-To get a list of available modules, keywords, symbols, or topics, type
-"modules", "keywords", "symbols", or "topics".  Each module also comes
-with a one-line summary of what it does; to list the modules whose name
-or summary contain a given string such as "spam", type "modules spam".
+To quit this help utility and return to the interpreter,
+type "q" or "quit".
```
Comment (zware) [inline Lib/pydoc.py]: I think we should replace "type" with "enter" in every case here.  The latter helps to clarify that you must hit the 'enter' key after your input; the former implies that is not necessary.

**Review by zware — APPROVED** [review]
One nit with the old wording, but otherwise this looks like a very reasonable rearrangement to me.

File: Lib/pydoc.py (hunk 1 above)
Comment (terryjreedy) [reply Lib/pydoc.py]: Agree.  3 replacements

File: Lib/pydoc.py [hunk 2]
```diff
@@ -2074,19 +2074,21 @@ def help(self, request, is_cli=False):
 
     def intro(self):
         self.output.write('''
-Welcome to Python {0}'s help utility!
-
-If this is your first time using Python, you should definitely check out
-the tutorial on the internet at https://docs.python.org/{0}/tutorial/.
+Welcome to Python {0}'s help utility! If this is your first time using
+Python, you should definitely check out the tutorial on the internet at
```
Comment (hugovk) [inline Lib/pydoc.py]: "on the internet at [URL]" is somewhat redundant. 

For brevity:
```suggestion
Python, you should definitely check out the tutorial at
```

Comment (terryjreedy) [reply Lib/pydoc.py]: Kind of quaint these days ;-).

**Review by hugovk — APPROVED** [review]

**Comment by terryjreedy** [comment]
Thank you both reviewers for helping to polish the text beyond simply moving it around.

**Pull request merged**